docs/pages/admin-guides/access-controls/access-request-plugins/ssh-approval-datadog.mdx

```diff
@@ -0,0 +1,381 @@
+---
+title: Access Requests with Datadog Incident Management (Self-Hosted)
+description: How to set up Teleport's Datadog Incident Management plugin for privilege elevation approvals.
+---
+
+## How it works
+
+With Teleport's Datadog Incident Management integration, engineers can access the
+infrastructure they need to resolve incidents without longstanding admin
+permissions that can become a vector for attacks.
+
+Teleport's Datadog Incident Management integration allows you to treat Teleport
+Role Access Requests as Datadog incidents, notify the appropriate on-call team,
+and approve or deny the requests via Teleport.
+
+This guide will explain how to set up Teleport's Access Request plugin for Datadog.
+
+![Datadog plugin architecture](../../../../img/enterprise/plugins/datadog/diagram.png)
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/machine-id/plugin-prerequisites.mdx!)
+
+- This requires the [`jq` CLI tool](https://jqlang.github.io/jq/).
+
+- A Datadog account with the role "Datadog Admin Role". The admin role is required to
+  create a Service Account and generate required credentials for the plugin.
+
+  You can see your role by visiting your user page in Datadog and navigating to
+  **Personal Settings -> Profile** and checking the value of the **Roles** field.
+
+- Either a Linux host or Kubernetes cluster where you will run the Datadog plugin.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/9. Create Datadog team
+
+To demonstrate the Datadog Incident Management plugin, create a team in Datadog.
+In the Datadog web UI, navigate to the **Teams** tab and click on **New Team** to
+create a new Datadog team. Name it "Teleport Access".
+
+We will configure the Datadog Incident Management plugin to route certain access
+requests to the "Teleport Access" team.
```

Comment:
Some of these steps are exactly the same between hosted and self-hosted guides, can we move them out into includes?